tests/jobs/localisation/test_integration.py

```diff
@@ -57,6 +56,12 @@ def test_localisation(setup_ert, obs_group_add, param_group_add, expected, snaps
     snapshot.assert_match(
         str(ert.update_configuration.dict()), "localisation_configuration"
     )
+    assert [
```

Comment:
@oyvindeide , could you have a peek at this added `assert`, does it make sense to include, or should the argument to the test be dropped?

Reply:
The `expected` parameter has never been used in this test. I think it makes more sense to just remove it